src/components/FooterFiltering/FooterFiltering.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filter: string;
+  setFilter: (a: string) => void;
+  todos: Todo[];
+};
+
+export const FooterFiltering: React.FC<Props> = ({
```

Comment:
just` Footer` (preferable since the parent jsx element you return here is footer), or just `Filter`